puppeteer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:13.8-slim
+
+RUN  apt-get update \
+    && apt-get install -y ca-certificates \
+     && apt-get install -y gnupg2 -y \
+     && apt-get install -yq libgconf-2-4 \
+     && apt-get install -y wget --no-install-recommends \
+     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+     && apt-get update \
+     && apt-get install -y google-chrome-stable --no-install-recommends \
+     && rm -rf /var/lib/apt/lists/* \
+     && wget --quiet https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh -O /usr/sbin/wait-for-it.sh \
+     && chmod +x /usr/sbin/wait-for-it.sh
+
+ADD package.json package-lock.json /
+
+RUN npm install
```

Comment:
I don't think we should add the package file here.  Instead I think we should simply have
```dockerfile
RUN npm install puppeteer
```
(as described on the [puppeteer web page](https://pptr.dev/).)